fuzz_lightyear/usage.py

```diff
@@ -93,6 +100,30 @@ def parse_args(argv: Optional[List[str]] = None) -> argparse.Namespace:
 
 def _is_valid_schema(path: str) -> Dict[str, Any]:
     _is_valid_path(path)
+
+    if path.endswith('json'):
+        return _is_json_schema(path)
+    elif path.endswith(('yaml', 'yml')):
+        return _is_yaml_schema(path)
+    else:
+        # try both json and yaml then raise if both file types
+        # don't work
+        try:
+            return _is_json_schema(path)
+        except argparse.ArgumentTypeError:
+            pass
+
+        try:
+            return _is_yaml_schema(path)
+        except argparse.ArgumentTypeError:
+            pass
+
+        raise argparse.ArgumentTypeError(
+            'Invalid schema file: {}'.format(path),
```

Comment:
nit: does it make sense to make this a bit more descriptive? (i.e. say that it doesn't appear to be either `JSON` or `YAML`?)

Reply:
I'll add the name of the exception to the error message to make debugging easier.

Reply:
Ok 35 seconds later I already regret that comment, your suggestion sounds great 👍